girder/esimmon_plugin/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup, find_packages
+
+setup(
+    name='esimmon-plugin',
+    version='0.0.1',
+    description='Finds items matching search parameters.',
```

Comment:
Isn't this plugin someone generic? If it not tired to our use case we might want to give it a more generic name?

Reply:
Updated to `Provide endpoints to support eSimMon dashboard.`